package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dydxprotocol/v4-localization",
-  "version": "1.1.37",
+  "version": "1.1.39",
```

Comment:
I think the manual change to 1.1.38 and then running the script bumped it to 1.1.39.

Reset to 1.1.37 and run bump verison